assets/js/buttons-examples.js

```diff
@@ -833,7 +846,18 @@ $(document).ready(function () {
           totalClasses += ` ${thisButtonClasses}`;
           totalWrittenClasses += ` .${thisButtonClasses}`;
         }
-
+        if(buttonArr[0].type=="blob")
```

Comment:
What's the purpose of this code? This button needs to be CSS only so please explain this code.

Reply:
In the" sButtons " web page the code in buttons-examples.js file automatically  inserts the ".sbtn "class to all the buttons which in turn interferes or overshadows or overwrites the properties written  in  button specific  classes(eg. .blob-btn).
So here I just stopped it from adding the sbtn class to the  div with " blob-btn" class in the page. It works fine for all the rest of the buttons . There is no change in them . This change is blob button specific , would cause no error in the rest of the buttons...
As per the above remarks I would make the necessary changes and introduce a new PR soon..
Thank You.

Reply:
Thank you for explaining this. However, I think this will cause confusion, as we need the `sbtn` to be the base class throughout the project. You can look at bootstrap for example and how they use `btn` as a base class for their buttons. If there are things in your button that you need to overwrite the properties of the base class `btn`, then try to use `!important` and please remove this code. Thank you.

Reply:
Okay...